data/src/main/java/com/shopapp/data/impl/CategoryRepositoryImpl.kt

```diff
@@ -17,9 +17,9 @@ class CategoryRepositoryImpl(private val api: Api) : CategoryRepository {
         }
     }
 
-    override fun getCategoryList(perPage: Int, paginationValue: String?): Single<List<Category>> {
+    override fun getCategoryList(perPage: Int, paginationValue: String?, rootCategoryId: String?): Single<List<Category>> {
```

Comment:
Do we really need this for each request? Can we pass this param for `ShopifyApi` class?

Reply:
Lets rename it to parentCategoryId